### PR TITLE
feat: Verify email with app

### LINKED ISSUE #1926: Verify email with app
**Describe the feature you'd like**
Add the content provider to receive URL and send the token to the server to verify email. 

**Would you like to work on the issue?**

- [x] Yes
- [ ] No
- Other:

### PR DESCRIPTION
**Pull request by liveHarshit**
Fixes #1926 

<p align="center">
 
</p>

### CODE REVIEW

File: app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt [hunk 1]
```diff
@@ -103,6 +108,7 @@ class ProfileFragment : Fragment(), BottomIconDoubleClick {
             .nonNull()
             .observe(viewLifecycleOwner, Observer {
                 rootView.snackbar(it)
+                profileViewModel.syncProfile()
```
Comment (anhanh11001) [inline app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt]: Why this line?

Comment (liveHarshit) [reply app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt]: To update verification status in profile section by fetching profile from the server.

Comment (liveHarshit) [reply app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt]: Okay, it's duplicate. Removing.

Comment (liveHarshit) [reply app/src/main/java/org/fossasia/openevent/general/auth/ProfileFragment.kt]: Updated

File: app/build.gradle [hunk 2]
```diff
@@ -44,10 +44,10 @@ android {
         }
         debug {
             buildConfigField "String", "DEFAULT_BASE_URL", '"https://open-event-api-dev.herokuapp.com/v1/"'
-            buildConfigField "String", "FRONTEND_URL", '"https://fossasia.github.io/open-event-frontend/"'
+            buildConfigField "String", "FRONTEND_URL", '"https://open-event-fe.netlify/"'
```
Comment (iamareebjamal) [inline app/build.gradle]: Duplicate, just use FRONTEND_HOST

Comment (liveHarshit) [reply app/build.gradle]: Updated

File: app/build.gradle [hunk 3]
```diff
@@ -37,17 +37,15 @@ android {
...
             buildConfigField "String", "STRIPE_API_KEY", '"'+STRIPE_API_TOKEN+'"'
             resValue "string",  "FRONTEND_HOST", "eventyay.com"
         }
         debug {
             buildConfigField "String", "DEFAULT_BASE_URL", '"https://open-event-api-dev.herokuapp.com/v1/"'
-            buildConfigField "String", "FRONTEND_URL", '"https://fossasia.github.io/open-event-frontend/"'
             buildConfigField "String", "MAPBOX_KEY", '"'+MAPBOX_KEY+'"'
             buildConfigField "String", "STRIPE_API_KEY", '"'+STRIPE_API_TOKEN+'"'
-            resValue "string", "FRONTEND_HOST", "fossasia.github.io/open-event-frontend"
+            resValue "string", "FRONTEND_HOST", "open-event-fe.netlify.com"
```
Comment (iamareebjamal) [inline app/build.gradle]: Why not buildConfigField? It's static, it's constant

Comment (liveHarshit) [reply app/build.gradle]: Cannot use in manifest file for handle app link data.

Comment (iamareebjamal) [reply app/build.gradle]: OK

**Pull request merged**